src/Controllers/DocumentationPageController.php

```diff
@@ -130,16 +118,23 @@ public function getListData($list): ArrayList
             return $navData;
         }
         foreach ($list->childNodes as $child) {
-
             // we only care about 'li' children (not newlines etc - DOMDocument gives us all sorts of stuff)
             if ($child->nodeName === 'li') {
                 $data = [];
 
-                // get data from the (first) 'a' tag contained in this 'li'
-                if ($child->getElementsByTagName('a')->item(0)) {
+                // check here to see if the FIRST thing in this li is the label text (ie, not wrapped in an a tag)
+                $firstChildData = array_filter(explode(PHP_EOL, $child->firstChild->data)); // remove newlines etc
+                $nonLinkItemData = current($firstChildData); // will be null if this li contains only tags, such as <a>
+
+                if ($nonLinkItemData) {
+                    $data['Type'] = 'Item';
+                    $data['Title'] = $nonLinkItemData;
+                } elseif ($child->getElementsByTagName('a')->item(0)) {
+                    // get data from the (first) 'a' tag contained in this 'li'
                     $link = $child->getElementsByTagName('a')->item(0);
                     $data['Title'] = $link->nodeValue;
                     $data['Link'] = $link->getAttribute('href');
+                    $data['Type'] = 'Link';
```

Comment:
I feel like there's a template update that hasn't come along for the ride on this PR, because the `Type` param isn't really used, where I think it can be managed in the template to render it as not-a-link when it's just set to "Item".

Something less crude than, but effectively:
```html
            <% if $Type == 'Link' %>
                <a class="userguide__menu-link" href="$Link">$Title</a>
            <% else_if $Type == 'Item' %>
                <span>$Title</span>
            <% end_if %>
```

in `DocumentationPageMenu.ss`? I think the logic is sound, just missing a piece.